zlib-rs/src/allocate.rs

```diff
@@ -460,12 +460,12 @@ mod tests {
     #[test]
     fn test_allocate_zeroed() {
         #[cfg(feature = "rust-allocator")]
-        test_allocate_zeroed_help(Allocator::RUST);
+        test_allocate_zeroed_help(super::RUST);
```

Comment:
```suggestion
        test_allocate_zeroed_help(RUST);
```

and elsewhere in the `tests` module.